migration/incremental/019-jit-off.sql

```diff
@@ -0,0 +1,8 @@
+-- PG JIT compilation doesn't play nicely with TimescaleDB planner and can cause a huge 
```

Comment:
Do we have bug reports against timescaledb for the jit issues, can we link them here as well so that it will be easier to assess in future if the jit issues have been resolved?

Reply:
I linked one issue reported by Mat however I believe I found additional case (need still to confirm).

Reply:
So we don't have a definitive list of issues yet

Reply:
Github descriptions aren't reflected in the code, and aren't easy to find from the code. I would have preferred this to have been in the migration file.